test/Main.purs

```diff
@@ -17,26 +17,28 @@ import Test.QuickCheck.Gen (Gen, arrayOf, chooseInt)
 
 data TestRoute
   = Root
-  | Foo String Int String { a :: String, b :: Boolean }
-  | Bar { id :: String, search :: String }
-  | Baz String (Array String)
+  | Foo
+  | Bar String Int String { a :: String, b :: Boolean }
+  | Baz { id :: String, search :: String }
+  | Qux String (Array String)
```

Comment:
I originally put made `Qux` nullary and put it at the bottom but quickcheck was sometimes generating `"qux"` which would cause the tests to fail.

I know the order doesn't matter here but it does when using `sum`, and I thought it was best to keep them the same.